README.md

```diff
@@ -1,19 +1,289 @@
 # Cypress CircleCI Orb
```

Comment:
I think we lost some important structure here from the last current version. Like breaking these sections up with titles, like **Usage**, etc.